test/module-test.ts

```diff
@@ -36,7 +36,7 @@ describe('napajs/module', function () {
         it('napa module', () => {
             return napaZone.execute((thisFilePath: string) => {
                 var assert = require("assert");
-                var napaModule = require(thisFilePath + '/module/addon/build/simple-napa-addon.napa');
+                var napaModule = require(thisFilePath + '/../bin/simple-napa-addon.napa');
```

Comment:
sorry, I meant simple-addon.napa

---
In reply to: [133275193](https://github.com/Microsoft/napajs/pull/41#discussion_r133275193) [](ancestors = 133275193,133246133)